packages/server/src/campaign/campaign.service.ts

```diff
@@ -30,6 +30,14 @@ export class CampaignService {
     return this.campaignModel.findById(campaignId).exec();
   }
 
+  public async findAllWithOrganizer(organizerId: string) {
+    return await this.campaignModel.find({ 'organizer.id': organizerId }).exec();
+  }
+
+  public async findOneWithOrganizer(organizerId: string, campaignId: string) {
+    return await this.campaignModel.find({ 'organizer.id': organizerId, _id: campaignId }).exec();
```

Comment:
Let's omit the `await` keyword if we are returning the function

```suggestion
    return this.campaignModel.find({ 'organizer.id': organizerId, _id: campaignId }).exec();
```